cairo/src/Slay/Cairo.hs

```diff
@@ -116,6 +116,7 @@ instance RenderElement g (PrimCurve g) where
           (x + fst pEnd)   (y + snd pEnd)
     Cairo.moveTo (x + x1) (y + y1)
     curveThrough p1 p2 p3
+    Cairo.setLineWidth $ realToFrac width
```

Comment:
`realToFrac` no longer needed after I introduced Backpack

Reply:
What do you mean ? `setLineWidth` takes a Double as parameter but I don't see `Double` instance for `Unsigned` defined